From 4ad2b82f02940919076bbd75002408a5bfef6a80 Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Thu, 8 Mar 2018 21:19:05 +0000
Subject: [PATCH] OS-6747 make clean removes brand hooks Reviewed by: Patrick
 Mooney <patrick.mooney@joyent.com> Reviewed by: Jerry Jelinek
 <jerry.jelinek@joyent.com> Approved by: John Levon <john.levon@joyent.com>

---
 usr/src/lib/brand/bhyve/zone/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/lib/brand/bhyve/zone/Makefile b/usr/src/lib/brand/bhyve/zone/Makefile
index f778fda27a..d7e957072c 100644
--- a/usr/src/lib/brand/bhyve/zone/Makefile
+++ b/usr/src/lib/brand/bhyve/zone/Makefile
@@ -20,7 +20,7 @@ PROGS =		attach detach statechange uninstall
 PROG1 =		boot
 PROG2 =		bhhwcompat
 PROGS +=	$(PROG1) $(PROG2)
-CLEANFILES +=	$(PROGS)
+CLEANFILES +=	$(PROG1) $(PROG2)
 
 $(PROG1) :=	LDLIBS += -lnvpair
 $(PROG2) :=	CPPFLAGS = -I$(COMPAT)/freebsd -I$(CONTRIB)/freebsd \
-- 
2.21.0

